README.md

```diff
@@ -18,76 +58,141 @@ Then it merges these cells if the distance between two clusters' centers is lowe
 }
 ```
 Input parameters :
- - `field`: must be of type geo_point.
- - `zoom`: mandatory integer parameter between 0 and 20. It represents the zoom level used in the request to aggregate geo points.
- - `radius`: radius in pixel. It is used to compute a geohash precision then to merge cluters based on this distance. Default to `50`. 
- - `ratio`: ratio used to make a second merging pass. If the value is `0`, no second pass is made. Default to `2`. 
- - `extent`: Extent of the tiles. Default to `256`
+- `field`: must be of type [geo_point](https://www.elastic.co/guide/en/elasticsearch/reference/7.10/geo-point.html)
+- `zoom`: mandatory integer parameter between 0 and 25. It represents the zoom level used in the request to aggregate geo points
```

Comment:
I guess the level refers to the TMS zoom level ? https://docs.nearmap.com/pages/viewpage.action?pageId=11206698